code/obj/mining.dm

```diff
@@ -2151,6 +2151,12 @@ TYPEINFO(/obj/item/cargotele)
 
 #undef SILICON_POWER_COST_MOD
 
+/obj/item/cargotele/efficient
+	name = "Hedron cargo transporter"
+	desc = "A device for teleporting crated goods. It's modified a bit from the standard design, and boasts a 20% efficiency improvement on the label."
+	cost = 20
```

Comment:
Honestly think this could be buffed a bit, half-cost seems fine. Or even infinite-use (0 cost) maybe. The convenience of a 20% discount in power usage seems pretty minimal for completing a whole requisition contract, and generally the power cell in a cargo tele isn't a big balance thing- rechargers are plentiful enough in cargo that it's only really costing you like 10 seconds per recharge normally.

Reply:
Yeah that sounds good. Minor efficiency boost + faster teleporting.

Reply:
Did the thing!